recordedit/form.controller.js

```diff
@@ -356,6 +366,7 @@
                             ErrorService.catchAll(error);
                         });
                     } else {
+                        // TODO: rethrow error here
                         vm.showSubmissionError(response);
                     }
                 }).catch(function (error) {
```

Comment:
pass `catchAll()` fn as the function rather than defining an anonymous function that calls it.